bool start_stack_frame
float32 stack_sec
---
float32 radius